packages/compose/index.ts

```diff
@@ -77,7 +78,7 @@ export interface Stamp extends ComposableFactory {
  * @property {Object} [propertyDescriptors] ES5 Property Descriptors applied to object instances
  * @property {Object} [staticPropertyDescriptors] ES5 Property Descriptors applied to Stamps
  */
-export interface Descriptor {
+export interface Descriptor extends PropertyMap {
```

Comment:
How necessary is this change?

In my mind PropertyMap does not regulate key names and values.
Whereas Descriptor is a specific interface - regulates keys and values.

I would rather not add this if possible. However, we are flexible here. Maybe I do not understand something.

Reply:
Another whoops. Fixed in next commit.